tests/unit/test_charm.py

```diff
@@ -7,14 +7,22 @@
 import hcl  # type: ignore[import-untyped]
 import ops
 import ops.testing
-from charm import VaultOperatorCharm, config_file_content_matches
+from charm import (
+    CHARM_POLICY_NAME,
+    CHARM_POLICY_PATH,
+    MACHINE_TLS_FILE_DIRECTORY_PATH,
+    VAULT_CHARM_APPROLE_SECRET_LABEL,
+    VAULT_DEFAULT_POLICY_NAME,
+    VaultOperatorCharm,
+    config_file_content_matches,
+)
 from charms.operator_libs_linux.v2.snap import Snap, SnapState
+from charms.vault_k8s.v0.vault_client import AuditDeviceType, Token, Vault
 from charms.vault_k8s.v0.vault_tls import CA_CERTIFICATE_JUJU_SECRET_LABEL
-from ops.model import ActiveStatus, WaitingStatus
+from ops.model import WaitingStatus
```

Comment:
For some statuses we import the status directly (WaitingStatus) and for others we call them from the imported `ops` (`ops.BlockedStatus`). We should likely standardize with 1 approach.

Reply:
Fixed in https://github.com/canonical/vault-operator/pull/62/commits/a3d4c74912a61f467e1f49c512b2d8f990c5d240